docs/customization/oauth.md

```diff
@@ -21,3 +21,18 @@ All the organization data will be used during the authorization process so the u
 Once you've created your application you'll get the settings to setup your client.
 
 Check [omniauth-decidim](https://github.com/decidim/omniauth-decidim) in order to configure your client application.
+
+## Performing more actions on omniauth registration
+
+Some times, there is the need to perform more actions than just ceating a user on registration, this is why `CreateOmniauthRegistration` command publishes a `"decidim.events.user.omniauth_registration` event after registration so that developers can subscribe to it and perform other actions like user verification or alike.
```

Comment:
Can we add a note saying that when you subscribe to these events it would be a good practice to handle them asynchronously in a job. If you subscribe to it and your service crashes the application it would also crash the user registration.

Maybe add a code example to handle the event?